app/src/main/java/com/itssuryansh/taaveez/presentation/ui/Notes.kt

```diff
@@ -0,0 +1,529 @@
+package com.itssuryansh.taaveez.presentation.ui
+
+import android.app.Activity
+import android.app.Dialog
+import android.content.Context
+import android.content.Intent
+import android.content.res.Configuration
+import android.graphics.Color
+import android.graphics.Typeface
+import android.os.Build
+import android.os.Bundle
+import android.text.*
+import android.util.Log
+import android.util.TypedValue
+import android.view.KeyEvent
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.*
+import androidx.activity.viewModels
+import androidx.annotation.RequiresApi
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.app.AppCompatDelegate
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.lifecycleScope
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.itssuryansh.taaveez.NotesApp
+import com.itssuryansh.taaveez.R
+import com.itssuryansh.taaveez.adapter.itemAdapter
+import com.itssuryansh.taaveez.data.database.NotesDao
+import com.itssuryansh.taaveez.databinding.ActivityNotesBinding
+import com.itssuryansh.taaveez.databinding.DeleteItemBinding
+import com.itssuryansh.taaveez.databinding.UpdateNotesBinding
+import com.itssuryansh.taaveez.domain.model.NotesEntity
+import com.itssuryansh.taaveez.presentation.NotesViewModel
+import com.itssuryansh.taaveez.utilities.Constants
+import dagger.hilt.android.AndroidEntryPoint
+import jp.wasabeef.richeditor.RichEditor
+import kotlinx.coroutines.launch
+import java.text.SimpleDateFormat
+import java.util.*
+
+@AndroidEntryPoint
+class Notes : AppCompatActivity() {
```

Comment:
Can you please add the suffix Activity to all activity files, 
Eg. NotesActivity

Reply:
Ok